run_tests.sh

```diff
@@ -13,4 +13,7 @@ if [ ! -z "$DNF_INSTALL" ]; then
     dnf -y --setopt=tsflags=nodocs --setopt=deltarpm=false install $DNF_INSTALL
 fi
 
+# Mark the current directory as safe for Git:
+git config --global --add safe.directory `pwd`
```

Comment:
This avoids backticks and also is more consistent with the rest of the script.

```suggestion
git config --global --add safe.directory $TEMP_DIR
```